### PR TITLE
Add FRIDAY HAUS playlist by Bleu Clair

### PR DESCRIPTION
**Pull request by maciejpedzich**
AKA, testing the entry validator bot... fingers crossed!

### CODE REVIEW

File: playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (mackorone) [inline playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj:1]: Did you intentionally introduce add a newline to this file? I think it'll just get removed the next time the script runs.

File: playlists/registry/6ZkpoSfbVoUoMm6rvrQEUm [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (mackorone) [inline playlists/registry/6ZkpoSfbVoUoMm6rvrQEUm:1]: Nice, it looks like your bot correctly validated this playlist ID!

**Review by mackorone — APPROVED** [review]

File: playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj (hunk 1 above)
Comment (maciejpedzich) [reply playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj:1]: No, I must have added it by accident. I'll fix it real quick

**Comment by maciejpedzich** [comment]
> Something went wrong while validating new entries! @mackorone should handle it shortly...

Hmm... weird, let me check the logs, because I think we've got a new edge case

**Comment by maciejpedzich** [comment]
@mackorone, I've implemented a fix. I'll check it out on my test repo. Feel free to merge this PR!

File: playlists/registry/6ZkpoSfbVoUoMm6rvrQEUm (hunk 2 above)
Comment (maciejpedzich) [reply playlists/registry/6ZkpoSfbVoUoMm6rvrQEUm:1]: Yup, it's so cool!

File: playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj (hunk 1 above)
Comment (mackorone) [reply playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj:1]: I might be missing something, but it looks like the newline is still there. Do you see it on your end?

Comment (maciejpedzich) [reply playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj:1]: > I might be missing something, but it looks like the newline is still there. Do you see it on your end?

I don't. Odd...

Comment (mackorone) [reply playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj:1]: Do you see the file as "changed" in this PR at least? If not, that'd be really weird...

Comment (maciejpedzich) [reply playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj:1]: Yes, it's marked as changed, but I can still see a single empty line

Comment (mackorone) [reply playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj:1]: > I can still see a single empty line as its content

This makes sense, actually. I believe the file was empty to start with, and now it contains a single empty line (terminated by a newline character). Truncating the file or reverting the change should fix the issue.

Comment (maciejpedzich) [reply playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj:1]: > > I can still see a single empty line as its content
> 
> This makes sense, actually. I believe the file was empty to start with, and now it contains a single empty line (terminated by a newline character). Truncating the file or reverting the change should fix the issue.

Will do! Also, not sure how the bot will react to this PR after new commits and changes to itself, so I'll close this PR and try again in a new one.

Comment (maciejpedzich) [reply playlists/registry/6JPMYu8YxGHBQ2Qv52JRaj:1]: @mackorone, so even if I remove the newline via the GitHub file editor, it seems to be added back anyway, with the commit being treated as empty-ish. In other words, changes have technically been made to the file, but in reality, they haven't been applied at all! I don't even remember modifying that file prior to today's PR in the first place...
